### PR TITLE
edits: changes to files in repo

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (5A11) [inline .github/workflows/ISSUE_TEMPLATE/config.yml:1]:
```suggestion
blank_issues_enabled: true
```

File: .github/workflows/PULL_REQUEST_TEMPLATE/release.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Release Details
+
+_[briefly describe the main changes with this release]_
+
+## Checklist
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I am making a pull request against the `master` branch
```
Comment (5A11) [inline .github/workflows/PULL_REQUEST_TEMPLATE/release.md:13]:
```suggestion
```

File: .github/workflows/PULL_REQUEST_TEMPLATE/release.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Checks and tests pass locally
+- [ ] I have updated the version in `pyproject.toml`
```
Comment (5A11) [inline .github/workflows/PULL_REQUEST_TEMPLATE/release.md:15]:
```suggestion
```

File: .github/workflows/PULL_REQUEST_TEMPLATE/release.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] I have regenerated and updated the API docs
```
Comment (5A11) [inline .github/workflows/PULL_REQUEST_TEMPLATE/release.md:16]:
```suggestion
```

File: DEVELOPING.md
Comment (5A11) [inline DEVELOPING.md:1]: delete this and put into a separate PR.

**Review by 5A11 — COMMENTED** [review]
- add code_owners and assign to aditya

File: .github/workflows/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
+name: 🐛 Bug Report
+description: Report a bug.
+labels: ["bug", "unconfirmed"]
```
Comment (5A11) [inline .github/workflows/ISSUE_TEMPLATE/bug-report.yml:3]:
```suggestion
labels: ["bug", "unconfirmed"]
assignees:
  - UrAvgDeveloper
```

File: .github/workflows/ISSUE_TEMPLATE/feature-request.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+name: ☝️Feature request
+description: Suggest an idea.
+labels: ["enhancement", "unconfirmed"]
```
Comment (5A11) [inline .github/workflows/ISSUE_TEMPLATE/feature-request.yml:3]:
```suggestion
labels: ["enhancement", "unconfirmed"]
assignees:
  - UrAvgDeveloper
```

File: .github/workflows/ISSUE_TEMPLATE/bug-report.yml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      label: Version
+      description: Which version were you using? Please enter the version number here.
+      placeholder: ex. v0.5.0
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment Details (Optional)
+      description: Provide any relevant information about your environment, such as operating system, Python version, and any other libraries used.
```
Comment (5A11) [inline .github/workflows/ISSUE_TEMPLATE/bug-report.yml:63]:
```suggestion
      description: Provide any relevant information about your environment, such as operating system and any other libraries used.
```

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
+- [ ] Documentation update.
+- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).
+
+## Checklist
+
+_Put an `x` in the boxes that apply:_
+
+- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
+- [ ] Checks and tests pass locally
```
Comment (5A11) [inline .github/pull_request_template.md:24]:
```suggestion
<!-- markdown-link-check-disable -->
- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
- [ ] Checks and tests pass locally
<!-- markdown-link-check-enable-->
```

File: .github/workflows/ISSUE_TEMPLATE/bug-report.yml [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+        - label: I checked the [existing issues](https://github.com/fetchai/fetch-wallet/issues) and made sure there are no similar bug reports
+          required: true
+  - type: dropdown
+    id: category
+    attributes:
+      label: Category
+      description: Select the category that best describes the bug.
+      options:
+        - Bug (unexpected behavior)
+        - Documentation Issue (error or unclear information in docs)
```
Comment (5A11) [inline .github/workflows/ISSUE_TEMPLATE/bug-report.yml:26]:
```suggestion
```

File: .github/pull_request_template.md (hunk 8 above)
Comment (5A11) [inline .github/pull_request_template.md:24]:
```suggestion
<!-- markdown-link-check-disable -->
- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
- [ ] Checks and tests pass locally
<!-- markdown-link-check-enable-->
```

File: .github/workflows/ISSUE_TEMPLATE/bug-report.yml [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+      label: Version
+      description: Which version were you using? Please enter the version number here.
+      placeholder: ex. v0.5.0
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment Details (Optional)
+      description: Provide any relevant information about your environment, such as operating system, protoc version, node version and any other libraries used.
```
Comment (5A11) [inline .github/workflows/ISSUE_TEMPLATE/bug-report.yml:64]:
```suggestion
      description: Provide any relevant information about your environment, such as operating system, bowser and its version, protoc version, node version and any other detail that might be useful.
```

**Review by sh-wallet — APPROVED** [review]
